android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java

```diff
@@ -121,16 +118,12 @@ private void startDeviceScan(@NonNull MethodCall call, Result result) {
                 new OnEventCallback<ScanResult>() {
                     @Override
                     public void onEvent(ScanResult data) {
-                        try {
-                            scanningStreamHandler.onScanResult(data);
-                        } catch (JSONException e) {
-                            e.printStackTrace();
-                        }
+                        scanningStreamHandler.onScanResult(data);
                     }
                 }, new OnErrorCallback() {
                     @Override
                     public void onError(BleError error) {
-                        Log.d("scanning_error", error.toString());
```

Comment:
Imho logging could be useful during development phase, but `Log.e` instead of `Log.d`

Reply:
I think we should log all errors on MBA side. It can be useful also in RN lib then.